index.js

```diff
@@ -76,6 +77,20 @@ module.exports = function (str, opts) {
 		urlObj.pathname = urlObj.pathname.replace(/\/{2,}/g, '/');
 	}
 
+	// remove directory index
+	if (opts.removeDirectoryIndex === true) {
+		opts.removeDirectoryIndex = [/^index/];
```

Comment:
Yes, agreed. Also, every time I make changes in response to a review is it always supposed to be in a new commit or is there a better or more correct way to do it ?